From d7c6701906b8ab2e40dbaa6d8fcf018e95259cfc Mon Sep 17 00:00:00 2001
From: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
Date: Sun, 19 Jan 2020 10:58:27 -0700
Subject: [PATCH 0917/1852] dt-bindings: mtd: Update dt binding documentation

commit f072e8cad757cf69a8f78f8cea6b5e71f6a5964e from
https://github.com/Xilinx/linux-xlnx.git

Added multi-die property to dt binding documentation.

Signed-off-by: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt b/Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt
index f03be904d3c2..51125e31e1e6 100644
--- a/Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt
+++ b/Documentation/devicetree/bindings/mtd/jedec,spi-nor.txt
@@ -78,6 +78,14 @@ Optional properties:
 		   cannot reboot properly if the flash is left in the "wrong"
 		   state. This boolean flag can be used on such systems, to
 		   denote the absence of a reliable reset mechanism.
+ - multi-die 	:  Some flash devices have multiple dies in it. Read operation
+		   in these devices is bounded by its die segment. In a
+		   continuous read, across multiple dies, when the last byte of
+		   the selected die segment is read, the next byte read is the
+		   first byte of the same die segment. So to handle this issue,
+		   split a read transaction, that spans across multiple banks,
+		   into one read per bank.This boolean flag can be used for such
+		   flash devices, to denote the presence of multiple dies.
 
 Example:
 
-- 
2.31.1

